membership.md

```diff
@@ -0,0 +1,170 @@
+# Community membership
+
+**Note:** This document is a work in progress
+
+This doc outlines the various responsibilities of contributor roles in Kubeflow. Kubeflow is divided into working groups that have stewardship over different subprojects/repositories
```

Comment:
Two lines above it says "document" here it is missing over 50 % of the word ;-)

Reply:
... or letters? 😛  just mentioning because it took me a sec to fully ack this comment is saying `s/doc/document/` :) 

```suggestion
This document outlines the various responsibilities of contributor roles in Kubeflow. Kubeflow is divided into working groups that have stewardship over different subprojects/repositories
```

Reply:
Heh, good catch.